functional_tests/login/01-timeout.yea

```diff
@@ -0,0 +1,8 @@
+id: 0.login.01-timeout
+plugin:
+  - wandb
+tag:
+  skips:
+    - platform: win
```

Comment:
just curious why?

Reply:
heh, I don't know actually, it was there already, I just move it to a separate yea file :)